Timer/MVTimerController.swift

```diff
@@ -10,7 +10,7 @@ class MVTimerController: NSWindowController {
     let mainView = MVMainView(frame: NSZeroRect)
 
     let window = MVWindow(mainView: mainView)
-    window.isReleasedWhenClosed = false
+    //window.isReleasedWhenClosed = false
```

Comment:
Why commenting instead of removing entirely?

Reply:
Oh, I was just keeping it around for reference while I tried out different memory handling strategies. But I've gone ahead and removed it now.